library/build.gradle

```diff
@@ -25,6 +25,10 @@ android {
         main.java.srcDirs += 'src/main/java'
         test.java.srcDirs += 'src/test/kotlin'
     }
+
+    testOptions {
+        unitTests.returnDefaultValues = true
```

Comment:
Added to prevent usage of `Log.d` from throwing "not implemented" exceptions while under test.